### PR TITLE
Docker Updates

### PR DESCRIPTION
**Pull request by ossarga**
- Updated Docker documentation to include commands to launch a Docker container without docker-compose.
- Fixed typos in the In-Memory documentation.
- Moved all the common services from the _common-services.yml_ into the _docker-compose.yml_ file.
- Added the `reaper-in-memory` service to docker-compose to launch a memory backed Reaper instance in a docker container.
- Removed `reaper-build-packages` from the main docker-compose file.
- Fixed broken Docker build command in Travis CI script.
- Changed links to use markdown instead of HTML.

### CODE REVIEW

**Comment by ossarga** [comment]
This branch needs to be squashed (and possibly rebased) prior to merging.

File: src/docs/content/docs/download/docker.md [hunk 1]
```diff
@@ -13,28 +13,99 @@ parent = "download"
...
+docker-compose build
+docker-compose run build
 ```
 
 ## Building Reaper Docker Image
 
-To build the Reaper Docker Image which is then added to the local image cache using the `cassandra-reaper:latest` tag, run the following commands from the top level directory:
+### Prerequisite
+
+The generation of the Docker image requires that the JAR file be built and placed in the _src/packages_ directory. If the JAR package is missing from the directory then it can built using either the steps in the [Docker package build](building-reaper-packages-with-docker) section (above), or in the <a href="../building#building-jars-from-source">Building from Source</a> section.
```
Comment (joaquincasares) [inline src/docs/content/docs/download/docker.md]: @ossarga can you do a proper Markdown link rather than href link?

**Review by joaquincasares — COMMENTED** [review]
Everything else lgtm

File: src/docs/content/docs/download/docker.md (hunk 1 above)
Comment (ossarga) [reply src/docs/content/docs/download/docker.md]: Unsure how to link across to a different section in the Markdown.

Comment (joaquincasares) [reply src/docs/content/docs/download/docker.md]: [Building from Source](../building#building-jars-from-source) should work.

```
[Building from Source](../building#building-jars-from-source) should work.
```

Comment (ossarga) [reply src/docs/content/docs/download/docker.md]: Hmmm ok. I probably specified it wrong when I was trying it. I have changed the link to use the markdown recommended in the Hugo docs
```
[Building from Source]({{< relref "building.md#building-jars-from-source" >}})
```

**Pull request merged**